src/v2/view-builder/views/ov/ChallengeAuthenticatorDataOktaVerifyView.js

```diff
@@ -22,6 +25,24 @@ const Body = SelectAuthenticatorVerifyViewBody.extend({
       }
     }];
   },
+
+  // If the `deviceKnown` attribute is true, we should put the signed_nonce method to the top of authenticator list.
```

Comment:
can we add another comment below this referencing what we do in ion-object-hander?
Something like
```
// This is in sync with v2/ion/ui-schema/ion-object-handler.js - createOVOptions
```

Reply:
I added the comment